internal/acceptance/sql_table_test.go

```diff
@@ -304,64 +304,69 @@ func constructManagedSqlTableTemplate(columnInfos []catalog.SqlColumnInfo) strin
 
 			%s
 			comment = "this table is managed by terraform"
-		}`, columnsTemplate)
+		}`, tableName, columnsTemplate)
 }
 
 func TestUcAccResourceSqlTable_RenameColumn(t *testing.T) {
 	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
 		skipf(t)("databricks_sql_table resource not available on GCP")
 	}
+	tableName := "rename"
```

Comment:
We should use random names, if in rare chances the test fails ( and the resource isn't destroyed) then this table will exists and the next run will have issue.